packages/app-builder-backend/src/entities/Author.ts

```diff
@@ -0,0 +1,13 @@
+import { ObjectType, Field, ID } from 'type-graphql'
```

Comment:
Tiny thing but kebab-case for files - it's not a big deal as it's only stub data so don't feel you have to change it